src/Components/WebAssembly/testassets/ThreadingApp/wwwroot/css/app.css

```diff
@@ -0,0 +1,181 @@
+html, body {
```

Comment:
I know it's extra work, but if possible it would be ideal to slim down this test app to eliminate all the unnecessary stuff. For example, it doesn't need any CSS, or the Bootstrap libraries, etc. It probably doesn't need layouts or a nav menu etc.

Reply:
That's easy. I just followed example of `StandaloneApp` in the same fonder.

Reply:
It's not a big deal. We just try to avoid accumulating more and more copies of the same boilerplate. Having one of them is useful as a playground during dev, but getting N of them means work to keep updating many copies of the same thing.

If you find it convenient to remove the unnecessary boilerplate from here that's great, but if not that OK too - we can trim it down some other time.